redux/actions.js

```diff
@@ -0,0 +1,37 @@
+import * as types from "./actionTypes";
+
+// Action Creators
+export function setCurrentUser(user) {
+    return {
+        type: types.SET_CURRENT_USER,
+        payload: user
+    };
+}
+
+export function setCurrentChat(chatId) {
+    return {
+        type: types.SET_CURRENT_CHAT,
+        payload: chatId
+    };
+}
+
+export function addMessage(id, cid, message, user, isSelf, emojis) {
+    return {
+        type: types.ADD_MESSAGE,
+        payload: { "id": id,
+                   "chats_id" : cid,
+                   "message": message,
+                   "user": user, 
+                   "isSelf": isSelf,
```

Comment:
this isn't relevant yet since we haven't done the messages, but isSelf is something we can just handle on the front end. i just added it on the chat UI so i know whether to put the message in a different color/side or not, but that can be determined if the message user is the same as the current user id. we won't need to store the data.

Reply:
Word. I think it can be removed for that reason from the redux actions though, if that's easy enough.

Reply:
Gotcha.